app/view/set-password.ejs

```diff
@@ -0,0 +1,49 @@
+<% include partials/head.ejs %>
+<title>Set password | Drinkerz</title>
+</head>
+
+<body>
+  <main class="sign-up__container">
+    <section class="sign-up__section">
+      <figure class="sign-up__header">
+        <img src="/static/media/images/logo.png" alt="logo Drinkerz">
+        <h1 class="sign-up__title">Drinkerz</h1>
+      </figure>
+
+      <form class="sign-up__form" action="/set-password" method="POST">
+        <p>
+          <%- user.username %>
+        </p>
+
+        <label>Password</label>
+        <input type="password" name="password" placeholder="password">
```

Comment:
Same as before (`<label>` Klik mij `<input type="text"></label>` Place the label around input fields so if the user clicks on it it will select the input fields)